packages/jsActions/web-nanoflow-actions/src/TakePicture.ts

```diff
@@ -23,38 +23,23 @@ export async function TakePicture(picture: mendix.lib.MxObject): Promise<boolean
     const getUserText = prepareLanguage();
 
     if (!picture) {
-        mx.ui.error(getUserText("Input parameter 'Picture' is required.", "Invoerparameter 'Afbeelding' is vereist. "));
-
-        return Promise.reject(false);
+        return Promise.reject(new Error("Input parameter 'Picture' is required."));
     }
 
     if (!picture.inheritsFrom("System.Image")) {
-        const entity = picture.getEntity();
-
-        mx.ui.error(
-            getUserText(
-                `Entity ${entity} does not inherit from 'System.Image'.`,
-                `Entiteit ${entity} erft niet van 'System.Image'. `
-            )
-        );
```

Comment:
Why are we getting rid of all these `mx.ui.error`?

Reply:
Internal errors should be visible to developers, unless is permission or some problem with the camera.